libs/community/langchain_community/document_loaders/notebook.py

```diff
@@ -104,29 +108,29 @@ def load(
         self,
     ) -> List[Document]:
         """Load documents."""
-        try:
-            import pandas as pd
-        except ImportError:
-            raise ImportError(
-                "pandas is needed for Notebook Loader, "
-                "please install with `pip install pandas`"
-            )
         p = Path(self.file_path)
 
         with open(p, encoding="utf8") as f:
             d = json.load(f)
 
-        data = pd.json_normalize(d["cells"])
```

Comment:
what does pd.json_normalize do

Reply:
From the [documentation](https://pandas.pydata.org/docs/reference/api/pandas.json_normalize.html): Normalizes semi-structured JSON data into a flat table.

Combined with `filtered_data = data[["cell_type", "source", "outputs"]]` this statement makes a table of [ cell_type, source, output ] information.

This behaviour can be simply replicated by a list of dictionaries with keys `cell_type`, `source`, `output`, without relying on a dependency.

Reply:
makes sense, could we add a simple unit test?

Reply:
@lachiewalker 👋 new to contributing here, I was wondering if I can help write the unit test?

Reply:
@paulonasc yeah, of course 🙂

Reply:
here's the commit for the unit tests btw! https://github.com/langchain-ai/langchain/pull/17721

Is it okay that it's in a separate PR?